File: src/main/java/jp/co/sunarch/apps/idprovider/model/oauth2/AccessTokenResponse.java
package jp.co.sunarch.apps.idprovider.model.oauth2;

import java.util.Set;

/**
 * @author takeshi
 */
public class AccessTokenResponse {

  private AccessToken accessToken;
  private AccessTokenType tokenType;
  private long expiresIn;
  private RefreshToken refreshToken;
  private Set<Scope> scope;

}

File: src/main/kotlin/jp/co/sunarch/apps/idprovider/model/oauth2/AccessTokenResponse.kt
package jp.co.sunarch.apps.idprovider.model.oauth2

/**
 * @author takeshi
 */
data class AccessTokenResponse(
    val accessToken: AccessToken,
    val tokenType: AccessTokenType,
    val expiresIn: Long?,
    val refreshToken: RefreshToken?,
    val scope: Set<Scope>?
)

